jujugui/static/gui/src/app/init/changes-utils.js

```diff
@@ -164,6 +164,22 @@ const changesUtils = {
   },
 
   /**
+    Convert an object to a keyval string e.g. {one: 'two', three: 'four'}
+    becomes one='two' three='four'.
```

Comment:
Please document what happens when the value is falsey.

Reply:
Done.